### PR TITLE
chore: add rule for detecting unused import in eslint

### PR DESCRIPTION
**Pull request by larsmoa**

### CODE REVIEW

File: viewer/.eslintrc.js [hunk 1]
```diff
@@ -14,7 +14,8 @@ module.exports = {
     ecmaVersion: 2020, // Allows for the parsing of modern ECMAScript features
     sourceType: 'module' // Allows for the use of imports
   },
-  plugins: ['header', '@typescript-eslint', 'jsdoc'],
+  plugins: ['header', '@typescript-eslint', 'jsdoc', 'unused-imports'],
```
Comment (maksnester) [inline viewer/.eslintrc.js:17]: Let’s set it as warn. It will be less annoying then, but precommit hook still will catch it

**Review by maksnester — APPROVED** [review]
LGTM but I suggest to use warn instead of error

**Comment by maksnester** [comment]
Meh, can’t see labels on pr from phone screen

File: viewer/.eslintrc.js (hunk 1 above)
Comment (larsmoa) [reply viewer/.eslintrc.js:17]: The thing is that compile will fail in CI if there are unused imports, @maksnester  - so I actually think it should be an error (if we don't change that in test compile).

**Pull request merged**